src/main/java/core/basesyntax/FigureSuplier.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSuplier {
+    private static final int NUMBER_COUNT = 10;
+    private static final int FIGURE_COUNT = 10;
+    private static final double DEAFAULT_RADIUS = 10.0;
+    private final ColorSuplier colorSuplier;
+    private Random random = new Random();
+
+    public FigureSuplier(ColorSuplier colorSuplier) {
+        this.colorSuplier = colorSuplier;
```

Comment:
1. keep 1 extra line between constants and fields
2. choose only one way to init fields - via constructor or inline
3. typo in `Suplier` (should be `Supplier`, so rename class too)
4. no need to pass dependency as constructor parameter, can be created just here
```suggestion

    private final ColorSupplier colorSupplier;
    private Random random;

    public FigureSuplier() {
        colorSupplier = new ColorSupplier();
        random = new Random();
```